source/integrations/zoom.rst

```diff
@@ -25,11 +25,15 @@ Zoom Setup Guide
 .. image:: ../images/zoom_system_console.png
 
 2. If you're using a self-hosted private cloud or on-premise Zoom server, enter the **Zoom URL** and **Zoom API URL** for the Zoom server, for example ``https://yourzoom.com`` and ``https://api.yourzoom.com/v2`` respectively. Leave blank if you're using Zoom's vendor-hosted SaaS service.
+
 3. Set the **API Key** and **API Secret**, generated by Zoom and used to create meetings and pull user data:
 
-  - Go to https://developer.zoom.us/me/ and log in.
-  - Switch to the **API** tab and click **Enable**, if not already enabled.
-  - Switch back to the **API** tab and copy the **API Key** and **API Secret**.
+  - Go to https://marketplace.zoom.us/ and log in.
+  - In the top left click on **Develop** and then **Build App**
+  - Enter a name for your app and select **Account-level App**, also disable **Intend to publish**.
```

Comment:
Propose starting a new sentence from `Also disable..`.

Also, `Intend to publish` --> `Intent to publish` I think?

Reply:
![screenshot from 2019-02-22 11-03-28](https://user-images.githubusercontent.com/7526550/53328164-a9ded800-38e9-11e9-9097-139f987ad3ba.png)
It's shortened for `Intend to publish this app on the Zoom marketplace`. Should this still be changed to Intent?

Reply:
How about:

```
Enter a name for your app and disable **Intend to publish this app on Zoom Marketplace**. Choose **Account-level app** as the app type.
```

This makes sure it's clear and obvious for someone reading through the instructions.

Reply:
Good Idea. Changed!